docs/content/en/docs/Architecture/network.md

```diff
@@ -2,24 +2,115 @@
 title: "P2P Network"
 linkTitle: "P2P Network"
 weight: 5
-date: 2022-11-13
+date: 2023-02-15
 description: >
+    How Kairos leverage Peer-to-peer (P2P) to self-coordinate clusters at the edge.
 ---
 
-Optionally Kairos supports full p2p connectivity.
+## Introduction
 
-Nodes automatically co-ordinates themselves and bootstrap from the ground-up using a distributed ledger:
+As more organizations seek to take advantage of the benefits of Kubernetes for their edge applications, the difficulties of managing large-scale clusters become apparent. Managing, configuring, and coordinating multiple clusters can be a complex and time-consuming process, especially when zero-touch configuration and self-coordination are necessary.
 
-![p2p](https://user-images.githubusercontent.com/2420543/195459436-236139cf-605d-4608-9018-ea80381d4e77.png)
+To address these challenges, Kairos provides an easy and robust solution for deploying Kubernetes workloads at the edge. By utilizing peer-to-peer (p2p) technology, Kairos can automatically coordinate and create Kubernetes clusters without requiring a control management interface. This frees users to concentrate on running and scaling their applications instead of spending time on cluster management.
 
-The P2P networking is used for co-ordinating and to bootstrap nodes, however, when this functionality is enabled there is also a distributed ledger accessible over the nodes that can be programmatically be accessed and used to store metadata.
+In this document, we will examine the advantages of using Kairos to deploy Kubernetes clusters at the edge, and how p2p technology facilitates self-coordination for a zero-touch configuration experience. We will also explore how Kairos' highly adaptable and container-based approach, combined with an immutable OS and meta-distribution, makes it an excellent choice for edge deployments.
 
-Kairos can automatically set up a VPN between the nodes by using a shared secret. This also allows the nodes to automatically coordinate, discover/configure and establish a network overlay spanning across multiple regions. [Edgevpn](https://github.com/mudler/edgevpn) is used for such purpose.
+## Overview: P2P for self-coordination
 
-The connection happens in three stages, where the discovery is driven by DHT (distributed hash table) and (multicast DNS) mDNS (which can be selectively disabled/enabled):
+<img align="right" width="200" src="https://user-images.githubusercontent.com/2420543/219048504-986da0e9-aca3-4c9e-b980-ba2a6dc03bf7.png">
 
-- Discovery
-- Gossip network
-- Full connectivity
+Kairos creates self-coordinated fully meshed clusters at the edge by using a combination of P2P technology, VPN, and Kubernetes. 
+
+This design is made up of several components:
+
+- the Kairos base OS with support for different distribution flavors and k3s combinations (see our support matrix [here](/docs/reference/image_matrix)).
```

Comment:
all other items don't have a dot at the end, feel free to remove this one or add to the others

```suggestion
- the Kairos base OS with support for different distribution flavors and k3s combinations (see our support matrix [here](/docs/reference/image_matrix))
```